portfolio/src/main/webapp/script.js

```diff
@@ -185,8 +188,8 @@ function createMapFunction() {
   document.head.appendChild(script);
 }
 
-function geocodeAddress(geocoder, resultsMap) {
-  var address = document.getElementById('address').value;
+function geocodeAddress(geocoder, resultsMap, address) {
+//   var address = document.getElementById('address').value;
```

Comment:
Maybe remove this line if it isn't needed?

Reply:
Fixed in following commit. Thanks!